### ISSUE TITLE
should library include babel directly?

### ISSUE THREAD

**Issue by phishy** [sprintly/sprintly-data#17]
I was writing a module and including sprintly-data as a dependency. I had mocha setup to include babel, which caused an error when sprintly-data tried to include it as well. Should it be included directly in a library?

 https://babeljs.io/docs/usage/require

**Comment by wookiehangover** [comment]
Good catch. We typically have been using this on the frontend and using the `browser` field in package.json to specify a different entry point.

I'll look into compiling the source and publishing a new release, but in the meantime you can bypass our call to babel by requiring the es6 entry point: `require('sprintly-data/sprintly-data')`
